KomodoPlatform/atomicDEX-Desktop
# Allow additional default coins

To test:
- do an assets reset. 
- KMD, LTC and BTC should be enabled by default
- You should not be able to disable KMD or LTC, but can disable and BTC
